docs/input/docs/installation/configure.md

```diff
@@ -0,0 +1,51 @@
+---
+Order: 20
+Title: Configuration
+Description: Configure BBT.StructureTools
+---
+
+The BBT Structure Tools are entirely configurable by code. This section covers the
+correct IoC container registration of the infrastructure components.
```

Comment:
Is IoC container a requirements for using this library? We should make this clear and use a clear language regarding this.

Reply:
Yes, it is - I've rewritten the sentence under the _Preconditions_ chapter to try to make this clear.